### DISCUSSION TITLE
Page APIs and TOC APIs need to be deprecated from Batch Service

### DISCUSSION THREAD

**Discussion by reshmi-nair** [Sunbird-Lern/Community#87] [Contribution]
Page APIs from sunbird-course-service, will be deprecated in coming releases. So adopters who is referring this API need to remove the reference.

Adopters who are using the TOC APIs from sunbird-course-service, needs to point to the Sunbird Knowlg content -service TOC APIs. ([Knowlg APIs](https://knowlg.sunbird.org/use/deprecations/release-5.2.0))

@rajeevsathish @vinukumar-vs @Krishnaj20 @swayangjit @rhwarrier

**Comment by rhwarrier** [comment]
@reshmi-nair  will this have any impact on older app versions etc.? Any action required on that front? 

+ [@vrayulu](https://github.com/vrayulu)  @pallakartheekreddy

**Reply by Krishnaj20** [reply]
@rhwarrier @reshmi-nair - I'm afraid ED may not be able to factor this in our 6.0.0 release. We can upgrade only in 7.0.0 (Dec '23). I hope support is provided for the older version of release.  

@vrayulu @vinukumar-vs @chitranshu-keshav
